privacy_evaluator/utils/model_utils.py

```diff
@@ -0,0 +1,67 @@
+from tensorflow import keras
+from torch import nn
+from art.estimators.estimator import BaseEstimator
+import numpy as np
+from copy import deepcopy
+from typing import Union
+import shutil
+
+
+def copy_and_reset_model(
+    model: Union[keras.Model, nn.Module, BaseEstimator]
+) -> Union[nn.Module, keras.Model]:
+    """
+    Get a clone of a model (i.e. with the same architecture) and reset the weights \
+        of this copy.
+
+    :param model: A TF or Torch model. Alternatively, an art-classifier encapsulating \
+        a TF or Torch model.
+    :return: The re-initialized copy of the given model. If `model` is a TF model \
```

Comment:
what do you mean by "the same applied in Torch case": do we also return a TF model then?

Reply:
Corrected to "If `model` is a torch model (or art-torch) then we return a torch model."